plugins/attach.py

```diff
@@ -0,0 +1,19 @@
+# Author: Fayas (https://github.com/FayasNoushad) (@FayasNoushad)
+
+from userge import userge, Message
+
+
+@userge.on_cmd("attach", about={
+    'header': "Attach any link in a message",
+    'usage': "{tr}attach [reply to a message]"})
+async def attach(update: Message):
+    '''Attach links in message''' 
+    if update.reply_to_message is None:
+        await update.reply_to_message.reply_text(
```

Comment:
`update.reply_to_message` is `None`. so you can't call `update.reply_to_message.reply_text`

Reply:
Sorry, its a mistake. Fixed.